### ISSUE TITLE
Does it account for the thrusters weight?

### ISSUE THREAD

**Issue by Bongobytes** [343N/spaceengineers-thrust-calc#2]
Does The Calculator Account for how much the thrsuters  weight?

**Comment by 343N** [comment]
It's been a while, but I believe it does, yes.

**Comment by aidanrlambert** [comment]
Yeah, it does. It kinda goes through an addative approach. It keeps adding both thruster force and thruster mass one thruster at a time until the force is enough to supply lift to the input mass + total thruster mass.
